EZInnocathon/EZInnocathon.csproj

```diff
@@ -189,14 +189,26 @@
     <Resource Include="ezo_logo_shadow_Tzu_icon.ico" />
   </ItemGroup>
   <ItemGroup>
+    <None Include="bin\Debug\Resource\crit.wav" />
     <Content Include="Resource\buttonsound %281%29.wav" />
```

Comment:
why is this not updated to resource too?

Reply:
@eduardopradana here

Reply:
bin\Debug\Resource itu folder yang dulu dicreate manual. skrg aku dah putus connectionnya ke folder itu jd skrg folder itu bisa dihapus saja. crit.wav nya skrg udah jadi Resource seperti di baris 196 dengan tag <Resource>